polynote-spark/src/main/scala/polynote/kernel/SparkPolyKernel.scala

```diff
@@ -128,7 +131,7 @@ class SparkPolyKernel(
   }
 
   override val init: IO[Unit] = super.init >> taskQueue.runTaskIO("spark", "Spark session", "Starting Spark session...") {
-    taskInfo => IO(session) >> IO.unit
+    taskInfo => IO(session).handleErrorWith(err => IO(logger.error(err)(err.getMessage)) *> IO.raiseError(err)) >> IO.unit
```

Comment:
I can remove these kinda things... was just trying to figure out why Spark was broken (it wasn't broken... just forgot that you don't get `spark` until you run a Scala cell 😞 )

Reply:
Doesn't hurt for Polynote to be chatty, we can always remove unnecessary logging when it gets to a more stable state.